packages/start-slicemachine/src/lib/setupSentry.ts

```diff
@@ -0,0 +1,70 @@
+import * as Sentry from "@sentry/node";
+
+import semver from "semver";
+import { App, H3Event, sendError } from "h3";
+import { SliceMachineManager } from "@slicemachine/manager";
+import { SENTRY_EXPRESS_DSN } from "../constants";
+
+export const setupSentry = async (
+	manager: SliceMachineManager,
+	app: App,
+): Promise<void> => {
+	!isDevEnv() || process.env.SENTRY_AUTH_TOKEN !== undefined;
+	if (!isSentryEnabled()) {
+		return;
+	}
+
+	const sliceMachineVersion =
+		await manager.versions.getRunningSliceMachineVersion();
+
+	let user;
+	try {
+		user = await manager.user.getProfile();
+	} catch {
+		// not logged in
+		// user stays undefined
+	}
+
+	const config = await manager.project.getSliceMachineConfig();
+
+	const isStableVersion =
+		/^\d+\.\d+\.\d+$/.test(sliceMachineVersion) &&
+		semver.lte("0.1.0", sliceMachineVersion);
+
+	Sentry.init({
+		dsn: process.env.SENTRY_EXPRESS_DSN ?? SENTRY_EXPRESS_DSN,
```

Comment:
Maybe this logic could be done in the constant file (similar to https://github.com/prismicio/slice-machine/blob/71f0fd828d77e43344970731cff0ce54ff81ee10/packages/manager/src/constants/API_TOKENS.ts, but simpler), this way if this token is used elsewhere, then it's consistent